autogpt_platform/backend/backend/data/graph.py

```diff
@@ -543,20 +548,40 @@ async def get_graph(
     where_clause: AgentGraphWhereInput = {
         "id": graph_id,
     }
+
     if version is not None:
         where_clause["version"] = version
-    elif not template:
+    else:
         where_clause["isActive"] = True
 
-    # TODO: Fix hack workaround to get adding store agents to work
-    if user_id is not None and not template:
-        where_clause["userId"] = user_id
-
     graph = await AgentGraph.prisma().find_first(
         where=where_clause,
         include=AGENT_GRAPH_INCLUDE,
         order={"version": "desc"},
     )
+
+    if not graph:
+        return None
```

Comment:
If we already have this, we can remove `if graph else None` in both returns of this function.

Reply:
graph could be None at this point as the return of `await AgentGraph.prisma().find_first(...)` is `AgentGraph.prisma | None`

I have this line in to prevent it crashing on the next line when I try an access a property of `graph`

Reply:
I meant this last one:
https://github.com/Significant-Gravitas/AutoGPT/pull/9179/commits/714913a7097e0b7e2267b0f542a36d049b3eec5e